app/src/main/java/com/example/lhi06/residue/MainActivity.java

```diff
@@ -19,6 +19,8 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    public static final String REVIEW_EXTRA = "com.example.lhi06.residue.REVIEW_EXTRA";
+
     private ReviewAdapter adapter;
     private List<AlbumReview> reviews;
```

Comment:
adapter and reviews variables can be declared locally in `onCreate()` and so don't need to be fields